index.html

```diff
@@ -36,7 +41,7 @@ <h1>I'm David <br>Glad to see you!</h1>
     </section>
     <section class="works">
```

Comment:
- [ OPTIONAL] If you look carefully, you will realize that the ordering of the projects on the mobile and desktop version are different. This is not a bug but a feature. You can try hitting that mark with the flexbox property **order**. Read more about it [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)